03-make/make_soup/makefile

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/make -f
+
+#variables for hirerarchy
+KITCHEN = kitchen
+KNIFE = ${KITCHEN}/knife
+TABLE = ${KITCHEN}/table
+POT = ${TABLE}/pot
+SPOON = ${TABLE}/spoon
+DINNING_ROOM = dinning-room
+FOOD_MARKET = food-market
+
+#files with goods
+POTATO = potato
+WATER = water
+CARROT = carrot
+
+#commands
+CREATE_DIR = -mkdir
```

Comment:
what is the reason of ignoring exit status of mkdir
In some cases it is better to use option mkdir -p

Reply:
done